NEWS.md

```diff
@@ -1,3 +1,7 @@
+# CHANGES IN GGIR VERSION 3.0-10
+
+- Day segmentation: Fix minor bugs in conversion of activity log timestamps #1090 and #1093.
```

Comment:
These should be two different items. The first one is not a bug but an enhancement. Can you rephrase as:
- Part 2 and 5: qwindow functionality enhanced to also consider fractions of minutes, `#1093`
- Part 2 and 5: Fix minor bug in activity diary loading when date format is %Y-%m-%d, `#1090`